integration/test/basic/basic_test.go

```diff
@@ -22,6 +23,21 @@ func TestChartLifecycle(t *testing.T) {
 	const testRelease = "tb-release"
 	const cr = "chart-operator-resource"
 
+	charts := []chart.Chart{
```

Comment:
This is now configured in the actual test instead of `setup` because it is directly related to the `fixtures` of the test.